code/modules/reagents/reagent_dispenser.dm

```diff
@@ -76,6 +76,13 @@
 		..()
 		reagents.add_reagent("water",1000)
 
+/obj/structure/reagent_dispensers/watertank/high/New()
+	name = "high-capacity water tank"
+	desc = "A highly-pressurized water tank made to hold vast amounts of water.."
+	icon_state = "watertank_high"
+		..()

```

Comment:
"code/modules/reagents/reagent_dispenser.dm:83:error: : invalid expression"

Might move New() from /watertank/high to before `..()` ?

Reply:
This ..() should be back one tab, as should the thing below it.
And the add_reagent() call should probably only add 4000, instead of 5000, unless you want the big tanks to have 5000.